.circleci/config.yml

```diff
@@ -1,47 +1,90 @@
 version: 2
-
 jobs:
-  build:
-    working_directory: ~/rosalind
+  test:
     docker:
-      - image: artsy/ruby:2.6.2-node-chrome
-        environment:
-          RAILS_ENV: test
-      - image: circleci/postgres:10.8-alpine
+      - image: artsy/hokusai:0.5
     steps:
+      - add_ssh_keys
       - checkout
-
-      - type: cache-restore
-        key: rosalind-bundler-{{ checksum "Gemfile.lock" }}
-      - run: bundle install --path vendor/bundle
-      - type: cache-save
-        key: rosalind-bundler-{{ checksum "Gemfile.lock" }}
-        paths:
-          - vendor/bundle
-      - run: yarn install
-
-      - run: mv config/database{.circle,}.yml
-      - run: bundle exec rake db:create db:migrate
-
-      - run: bundle exec rake webpacker:compile
-
-      - run: bundle exec rubocop
-      - run: yarn run lint
-
-      - run:
-          name: Browser testing version info
-          command: |
-            google-chrome --version
-
-      - run: bundle exec rspec
-      - run: yarn run test -- --runInBand
-
-      # - run: bundle exec danger
-
-      - run: bash .circleci/heroku_setup
-      - add_ssh_keys:
-          fingerprints:
-            - "d1:68:35:96:47:c2:de:36:dd:9f:e4:90:84:be:2e:63"
-      - deploy:
+      - setup_remote_docker
+      - run:
+          name: Test
+          command: hokusai test
+  push:
```

Comment:
We should use the shared Orbs for these steps when possible: https://github.com/artsy/orbs/tree/master/src/hokusai